### ISSUE TITLE
2025-02-12, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#81]
# Today's Arxiv Papers

(1) [DarwinLM: Evolutionary Structured Pruning of Large Language Models](http://arxiv.org/abs/2502.07780v1), 2025-02-11

(2) [Auditing Prompt Caching in Language Model APIs](http://arxiv.org/abs/2502.07776v1), 2025-02-11

(3) [Automatic Robot Task Planning by Integrating Large Language Model with Genetic Programming](http://arxiv.org/abs/2502.07772v1), 2025-02-11

(4) [Breaking Down Bias: On The Limits of Generalizable Pruning Strategies](http://arxiv.org/abs/2502.07771v1), 2025-02-11

(5) [Great Power Brings Great Responsibility: Personalizing Conversational AI for Diverse Problem-Solvers](http://arxiv.org/abs/2502.07763v1), 2025-02-11

(6) [Scalable Fingerprinting of Large Language Models](http://arxiv.org/abs/2502.07760v1), 2025-02-11

(7) [Towards Efficient Optimizer Design for LLM via Structured Fisher Approximation with a Low-Rank Extension](http://arxiv.org/abs/2502.07752v1), 2025-02-11

(8) [WHODUNIT: Evaluation benchmark for culprit detection in mystery stories](http://arxiv.org/abs/2502.07747v1), 2025-02-11

(9) [The Economics of Large Language Models: Token Allocation, Fine-Tuning, and Optimal Pricing](http://arxiv.org/abs/2502.07736v1), 2025-02-11

(10) [Economics of Sourcing Human Data](http://arxiv.org/abs/2502.07732v1), 2025-02-11

(11) [Verifying LLM-Generated Code in the Context of Software Verification with Ada/SPARK](http://arxiv.org/abs/2502.07728v1), 2025-02-11

(12) [MAGELLAN: Metacognitive predictions of learning progress guide autotelic LLM agents in large goal spaces](http://arxiv.org/abs/2502.07709v1), 2025-02-11

(13) [A Framework for LLM-powered Design Assistants](http://arxiv.org/abs/2502.07698v1), 2025-02-11

(14) [Large Language Models as Proxies for Theories of Human Linguistic Cognition](http://arxiv.org/abs/2502.07687v1), 2025-02-11

(15) [Auto-Drafting Police Reports from Noisy ASR Outputs: A Trust-Centered LLM Approach](http://arxiv.org/abs/2502.07677v1), 2025-02-11

(16) [SymGPT: Auditing Smart Contracts via Combining Symbolic Execution with Large Language Models](http://arxiv.org/abs/2502.07644v1), 2025-02-11

(17) [FoQA: A Faroese Question-Answering Dataset](http://arxiv.org/abs/2502.07642v1), 2025-02-11

(18) [Goedel-Prover: A Frontier Model for Open-Source Automated Theorem Proving](http://arxiv.org/abs/2502.07640v1), 2025-02-11

(19) [Exploring Mobile Touch Interaction with Large Language Models](http://dx.doi.org/10.1145/3706598.3713554), 2025-02-11

(20) [Beyond Prompting: Time2Lang -- Bridging Time-Series Foundation Models and Large Language Models for Health Sensing](http://arxiv.org/abs/2502.07608v1), 2025-02-11

(21) [Towards Zero-Shot Anomaly Detection and Reasoning with Multimodal Large Language Models](http://arxiv.org/abs/2502.07601v1), 2025-02-11

(22) [Towards spatial computing: recent advances in multimodal natural interaction for XR headsets](http://arxiv.org/abs/2502.07598v1), 2025-02-11

(23) [We Can't Understand AI Using our Existing Vocabulary](http://arxiv.org/abs/2502.07586v1), 2025-02-11

(24) [PIM Is All You Need: A CXL-Enabled GPU-Free System for Large Language Model Inference](http://dx.doi.org/10.1145/3676641.3716267), 2025-02-11

(25) [JBShield: Defending Large Language Models from Jailbreak Attacks through Activated Concept Analysis and Manipulation](http://arxiv.org/abs/2502.07557v1), 2025-02-11

(26) [O1 Embedder: Let Retrievers Think Before Action](http://arxiv.org/abs/2502.07555v1), 2025-02-11

(27) [Grammar Control in Dialogue Response Generation for Language Learning Chatbots](http://arxiv.org/abs/2502.07544v1), 2025-02-11

(28) [Harnessing Language's Fractal Geometry with Recursive Inference Scaling](http://arxiv.org/abs/2502.07503v1), 2025-02-11

(29) [LLM-Sketch: Enhancing Network Sketches with LLM](http://arxiv.org/abs/2502.07495v1), 2025-02-11

(30) [Mask-Enhanced Autoregressive Prediction: Pay Less Attention to Learn More](http://arxiv.org/abs/2502.07490v1), 2025-02-11

(31) [Improving Adaptive Moment Optimization via Preconditioner Diagonalization](http://arxiv.org/abs/2502.07488v1), 2025-02-11

(32) [Multi-Agent Collaboration for Multilingual Code Instruction Tuning](http://arxiv.org/abs/2502.07487v1), 2025-02-11

(33) [ETimeline: An Extensive Timeline Generation Dataset based on Large Language Model](http://arxiv.org/abs/2502.07474v1), 2025-02-11

(34) [JamendoMaxCaps: A Large Scale Music-caption Dataset with Imputed Metadata](http://arxiv.org/abs/2502.07461v1), 2025-02-11

(35) [Logarithmic Regret for Online KL-Regularized Reinforcement Learning](http://arxiv.org/abs/2502.07460v1), 2025-02-11

(36) [PerCul: A Story-Driven Cultural Evaluation of LLMs in Persian](http://arxiv.org/abs/2502.07459v1), 2025-02-11

(37) [Forget What You Know about LLMs Evaluations - LLMs are Like a Chameleon](http://arxiv.org/abs/2502.07445v1), 2025-02-11

(38) [Approximating Human Strategic Reasoning with LLM-Enhanced Recursive Reasoners Leveraging Multi-agent Hypergames](http://arxiv.org/abs/2502.07443v1), 2025-02-11

(39) [RomanLens: Latent Romanization and its role in Multilinguality in LLMs](http://arxiv.org/abs/2502.07424v1), 2025-02-11

(40) [Entity Linking using LLMs for Automated Product Carbon Footprint Estimation](http://arxiv.org/abs/2502.07418v1), 2025-02-11

(41) [EgoTextVQA: Towards Egocentric Scene-Text Aware Video Question Answering](http://arxiv.org/abs/2502.07411v1), 2025-02-11

(42) [On Iterative Evaluation and Enhancement of Code Quality Using GPT-4o](http://arxiv.org/abs/2502.07399v1), 2025-02-11

(43) [FinRL-DeepSeek: LLM-Infused Risk-Sensitive Reinforcement Learning for Trading Agents](http://arxiv.org/abs/2502.07393v1), 2025-02-11

(44) [Target-Augmented Shared Fusion-based Multimodal Sarcasm Explanation Generation](http://arxiv.org/abs/2502.07391v1), 2025-02-11

(45) [LLMs Can Easily Learn to Reason from Demonstrations Structure, not content, is what matters!](http://arxiv.org/abs/2502.07374v1), 2025-02-11

(46) [EvoFlow: Evolving Diverse Agentic Workflows On The Fly](http://arxiv.org/abs/2502.07373v1), 2025-02-11

(47) [LongReD: Mitigating Short-Text Degradation of Long-Context Large Language Models via Restoration Distillation](http://arxiv.org/abs/2502.07365v1), 2025-02-11

(48) [Bridging the Evaluation Gap: Leveraging Large Language Models for Topic Model Evaluation](http://arxiv.org/abs/2502.07352v1), 2025-02-11

(49) [KABB: Knowledge-Aware Bayesian Bandits for Dynamic Expert Coordination in Multi-Agent Systems](http://arxiv.org/abs/2502.07350v1), 2025-02-11

(50) [BenchMAX: A Comprehensive Multilingual Evaluation Suite for Large Language Models](http://arxiv.org/abs/2502.07346v1), 2025-02-11

(51) [Aligning Large Language Models to Follow Instructions and Hallucinate Less via Effective Data Filtering](http://arxiv.org/abs/2502.07340v1), 2025-02-11

(52) [Semantic to Structure: Learning Structural Representations for Infringement Detection](http://arxiv.org/abs/2502.07323v1), 2025-02-11

(53) [MEMIT-Merge: Addressing MEMIT's Key-Value Conflicts in Same-Subject Batch Editing for LLMs](http://arxiv.org/abs/2502.07322v1), 2025-02-11

(54) [CodeI/O: Condensing Reasoning Patterns via Code Input-Output Prediction](http://arxiv.org/abs/2502.07316v1), 2025-02-11

(55) [Prompt-Based Document Modifications In Ranking Competitions](http://arxiv.org/abs/2502.07315v1), 2025-02-11

(56) [CreAgent: Towards Long-Term Evaluation of Recommender System under Platform-Creator Information Asymmetry](http://arxiv.org/abs/2502.07307v1), 2025-02-11

(57) [TRAVEL: Training-Free Retrieval and Alignment for Vision-and-Language Navigation](http://arxiv.org/abs/2502.07306v1), 2025-02-11

(58) [Small Language Model Makes an Effective Long Text Extractor](http://arxiv.org/abs/2502.07286v1), 2025-02-11

(59) [GENERator: A Long-Context Generative Genomic Foundation Model](http://arxiv.org/abs/2502.07272v1), 2025-02-11

(60) [When More is Less: Understanding Chain-of-Thought Length in LLMs](http://arxiv.org/abs/2502.07266v1), 2025-02-11

(61) [NARCE: A Mamba-Based Neural Algorithmic Reasoner Framework for Online Complex Event Detection](http://arxiv.org/abs/2502.07250v1), 2025-02-11

(62) [DrugImproverGPT: A Large Language Model for Drug Optimization with Fine-Tuning via Structured Policy Optimization](http://arxiv.org/abs/2502.07237v1), 2025-02-11

(63) [Graph RAG-Tool Fusion](http://arxiv.org/abs/2502.07223v1), 2025-02-11

(64) [A Memory Efficient Randomized Subspace Optimization Method for Training Large Language Models](http://arxiv.org/abs/2502.07222v1), 2025-02-11

(65) [MLLM4PUE: Toward Universal Embeddings in Computational Pathology through Multimodal LLMs](http://arxiv.org/abs/2502.07221v1), 2025-02-11

(66) [LUNAR: LLM Unlearning via Neural Activation Redirection](http://arxiv.org/abs/2502.07218v1), 2025-02-11

(67) [Provably Efficient RLHF Pipeline: A Unified View from Contextual Bandits](http://arxiv.org/abs/2502.07193v1), 2025-02-11

(68) [Bag of Tricks for Inference-time Computation of LLM Reasoning](http://arxiv.org/abs/2502.07191v1), 2025-02-11

(69) [Understanding LLMs' Fluid Intelligence Deficiency: An Analysis of the ARC Task](http://arxiv.org/abs/2502.07190v1), 2025-02-11

(70) [A Large-Scale Benchmark for Vietnamese Sentence Paraphrases](http://arxiv.org/abs/2502.07188v1), 2025-02-11

(71) [Perceived Confidence Scoring for Data Annotation with Zero-Shot LLMs](http://arxiv.org/abs/2502.07186v1), 2025-02-11

(72) [Refine Knowledge of Large Language Models via Adaptive Contrastive Learning](http://arxiv.org/abs/2502.07184v1), 2025-02-11

(73) [Don't Just Demo, Teach Me the Principles: A Principle-Based Multi-Agent Prompting Strategy for Text Classification](http://arxiv.org/abs/2502.07165v1), 2025-02-11

(74) [Does Training on Synthetic Data Make Models Less Robust?](http://arxiv.org/abs/2502.07164v1), 2025-02-11

(75) [Rethinking Fine-Tuning when Scaling Test-Time Compute: Limiting Confidence Improves Mathematical Reasoning](http://arxiv.org/abs/2502.07154v1), 2025-02-11

(76) [Ask Patients with Patience: Enabling LLMs for Human-Centric Medical Dialogue with Grounded Reasoning](http://arxiv.org/abs/2502.07143v1), 2025-02-11

(77) [Language-TPP: Integrating Temporal Point Processes with Language Models for Event Analysis](http://arxiv.org/abs/2502.07139v1), 2025-02-11

(78) [Interactive Data Harmonization with LLM Agents](http://arxiv.org/abs/2502.07132v1), 2025-02-10

(79) [Cardiverse: Harnessing LLMs for Novel Card Game Prototyping](http://arxiv.org/abs/2502.07128v1), 2025-02-10

(80) [Structural Reformation of Large Language Model Neuron Encapsulation for Divergent Information Aggregation](http://arxiv.org/abs/2502.07124v1), 2025-02-10

(81) [Online Scheduling for LLM Inference with KV Cache Constraints](http://arxiv.org/abs/2502.07115v1), 2025-02-10

(82) [Evaluating the Systematic Reasoning Abilities of Large Language Models through Graph Coloring](http://arxiv.org/abs/2502.07087v1), 2025-02-10

(83) [Multi-turn Evaluation of Anthropomorphic Behaviours in Large Language Models](http://arxiv.org/abs/2502.07077v1), 2025-02-10

(84) [IRepair: An Intent-Aware Approach to Repair Data-Driven Errors in Large Language Models](http://arxiv.org/abs/2502.07072v1), 2025-02-10

(85) [Specializing Large Language Models to Simulate Survey Response Distributions for Global Populations](http://arxiv.org/abs/2502.07068v1), 2025-02-10

(86) [Repository-level Code Search with Neural Retrieval Methods](http://arxiv.org/abs/2502.07067v1), 2025-02-10

(87) [Using Contextually Aligned Online Reviews to Measure LLMs' Performance Disparities Across Language Varieties](http://arxiv.org/abs/2502.07058v1), 2025-02-10

(88) [Tokenization Standards for Linguistic Integrity: Turkish as a Benchmark](http://arxiv.org/abs/2502.07057v1), 2025-02-10

(89) [Autonomous Deep Agent](http://arxiv.org/abs/2502.07056v1), 2025-02-10

(90) [Large Language Models in Software Security: A Survey of Vulnerability Detection Techniques and Insights](http://arxiv.org/abs/2502.07049v1), 2025-02-10

(91) [SnipGen: A Mining Repository Framework for Evaluating LLMs for Code](http://arxiv.org/abs/2502.07046v1), 2025-02-10

(92) [Scalable and Ethical Insider Threat Detection through Data Synthesis and Analysis by LLMs](http://arxiv.org/abs/2502.07045v1), 2025-02-10

(93) [Automated Consistency Analysis of LLMs](http://arxiv.org/abs/2502.07036v1), 2025-02-10

(94) [AIMS.au: A Dataset for the Analysis of Modern Slavery Countermeasures in Corporate Statements](http://arxiv.org/abs/2502.07022v1), 2025-02-10

(95) [Finding Words Associated with DIF: Predicting Differential Item Functioning using LLMs and Explainable AI](http://arxiv.org/abs/2502.07017v1), 2025-02-10

(96) [Demystifying Singular Defects in Large Language Models](http://arxiv.org/abs/2502.07004v1), 2025-02-10

(97) [SyncMind: Measuring Agent Out-of-Sync Recovery in Collaborative Software Engineering](http://arxiv.org/abs/2502.06994v1), 2025-02-10

(98) [Investigating the Zone of Proximal Development of Language Models for In-Context Learning](http://arxiv.org/abs/2502.06990v1), 2025-02-10

(99) [Position: Episodic Memory is the Missing Piece for Long-Term LLM Agents](http://arxiv.org/abs/2502.06975v1), 2025-02-10

(100) [Visual Agentic AI for Spatial Reasoning with a Dynamic API](http://arxiv.org/abs/2502.06787v1), 2025-02-10
